datacube/virtual/transformations.py

```diff
@@ -39,19 +41,14 @@ def selective_apply(data, apply_to=None, key_map=None, value_map=None):
 
 class MakeMask(Transformation):
     """
-    Create a mask that would only keep pixels for which the specified measurement
-    satisfies `flags` specification.
+    Create a mask that would only keep pixels for which the measurement with `mask_measurement_name`
+    of the `product` satisfies `flags`.
 
-    Alias in recipe: ``make_mask``.
```

Comment:
Can we have this line back please since the others seem to have them?

Reply:
np